pkg/ccl/logictestccl/testdata/logic_test/multi_region_privileges

```diff
@@ -232,7 +233,8 @@ ALTER DATABASE repartition_privs DROP REGION "us-east-1"
 user root
 
 statement ok
-GRANT CREATE ON repartition_privs.rbr TO testuser
+GRANT CREATE ON repartition_privs.rbr TO testuser;
+GRANT CREATE ON repartition_privs.rbr2 TO testuser
```

Comment:
Need to double check if this really makes sense, having to GRANT CREATE on all tables in the database to do `SET PRIMARY REGION`